(dsk_swift_v3dynamic) root@mm-fudan-chai-l20-1-0:/opt/nas/p/dongshaokang/AgentCPM-GUI# python /opt/nas/p/dongshaokang/AgentCPM-GUI/eval/eval_results/V2-Finish-Test/convert_729.py
Traceback (most recent call last):
  File "/opt/nas/p/dongshaokang/AgentCPM-GUI/eval/eval_results/V2-Finish-Test/convert_729.py", line 102, in <module>
    filter_template(input_file='/opt/nas/p/dongshaokang/AgentCPM-GUI/eval/eval_results/V1/CPT630/android_control_low_test_v1/all_copy.jsonl', output_file='/opt/nas/p/dongshaokang/AgentCPM-GUI/eval/eval_results/V1/CPT630/android_control_low_test_v1/all copy2.jsonl')
  File "/opt/nas/p/dongshaokang/AgentCPM-GUI/eval/eval_results/V2-Finish-Test/convert_729.py", line 92, in filter_template
    data = json.load(f)
  File "/opt/nas/p/conda/envs/dsk_swift_v3dynamic/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/opt/nas/p/conda/envs/dsk_swift_v3dynamic/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/opt/nas/p/conda/envs/dsk_swift_v3dynamic/lib/python3.10/json/decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 2 column 1 (char 18055)
(dsk_swift_v3dynamic) root@mm-fudan-chai-l20-1-0:/opt/nas/p/dongshaokang/AgentCPM-GUI# 
